README.md

```diff
@@ -1,79 +1,97 @@
+  - `sc-builder` loads information from SCs files into storage (SCs-code is a one of external representation languages of SC-code)
+
+<details>
+   <summary>More info</summary>
+
+sc-machine is a **platform-independent graph database management system** that can store / retrieve knowledge graphs and run tasks (agents) on them.
+
+Both declarative (data, data structures, documentation, tasks specification, etc.) and procedural
+(programs, modules, systems, communication between systems) knowledge is represented using the same language: SC-code.
+
+</details>
+
+## Documentation
+
+- A brief user manual and developer docs are hosted on our [GitHub Pages](https://ostis-ai.github.io/sc-machine).
+- Full documentation, including:
+  - core concepts
+  - rationale behind sc-machine
+  - system design
+  - software interfaces
+  
+  is redistributed in a form of [SCn-TeX document](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
+
+## Quick start
+
+sc-machine is a core of any ostis-system, so you can use a reference system named [OSTIS Web platform](https://github.com/ostis-ai/ostis-web-platform) to get sc-machine up and running quickly.
+
+## Installation
+
+- Docker:
+  We're providing a Docker image for this project. Head to [Installing with Docker](https://ostis-ai.github.io/sc-machine/docker) to learn more. It's the recommended way to deploy sc-machine.
+- Native:
+  If you do not have the option to deploy the system using Docker please refer to the docs for your OS:
+  [Build on Linux](https://ostis-ai.github.io/sc-machine/build/linux-build/) or [Build on macOS](https://ostis-ai.github.io/sc-machine/build/osx-build/) respectively.
+
+  Note: currently, sc-machine isn't *natively* supported on Windows.
+
+## Usage
+
+- Docker
+  
+  ```sh
+  # build kb
+  docker run --rm -it -v <full path to kb sources>:/kb -v sc-machine_kb:/kb.bin ostis/sc-machine:latest build /kb
+  # run sc-server
+  docker run --rm -it -v sc-machine_kb:/kb.bin ostis/sc-machine:latest serve
+  ```
+
+- Native
+
+  ```sh
+  #build kb
+  python3 scripts/build_kb.py -c config/sc-machine.ini <path to kb folder with SCs and SCg sources (or path to repo.path file)>
+  #launch sc-server
+  ./bin/sc-server -c config/sc-machine.ini
+  ```
+
+Most of these commands have a help page bundled inside, so if you have any questions or want to customize the command behavior, use `--help` flag to learn more about them.
 
 ## Config
 
-This repository provides a default configuration for sc-machine. To customize *sc-machine* usage you can 
-[create your own config file](https://ostis-ai.github.io/sc-machine/other/config).
+This repository provides a default configuration for sc-machine. To customize *sc-machine* to suit your needs you can [create your own config file](https://ostis-ai.github.io/sc-machine/other/config).
+
+## Feedback
+
+Contributions, bug reports and feature requests are welcome!
```

Comment:
```suggestion
Contributions, bug reports, and feature requests are welcome!
```

Reply:
Oh, Oxford comma? Classy :)
I assume our students may misinterpret it as a mistake (even though it's a stylistic/optional punctuation), so I'd choose to omit it in this case 🤔

Reply:
There are Oxford commas in one of your PR's, so I just propose to unificate such syntactic constructions)

Reply:
It looks like I've missed it then, it wasn't my intention. Can you send a link please?